### PR TITLE
Handle pagination

### LINKED ISSUE #13: Collect resources beyond the first page

### PR DESCRIPTION
**Pull request by elazarg**
Fix #13 

This is only tested vacuously, due to lack of permissions.

### CODE REVIEW

**Review by zivnevo — CHANGES_REQUESTED** [review]
Thanks @elazarg , the code looks great. Please see two tiny comments below.

I think we can consider the linter errors as false-positives. Feel free to disable with a proper [nolint](https://golangci-lint.run/usage/false-positives/#nolint-directive).

File: pkg/ibm/vpc_collection.go [hunk 1]
```diff
@@ -8,177 +8,304 @@ import (
 	"github.com/np-guard/cloud-resource-collector/pkg/ibm/datamodel"
 )
 
-// Get (the first page of) VPCs
+const pageSize = 4
```
[the remark this answers is not in the document]
Comment (kyorav) [reply pkg/ibm/vpc_collection.go:11]: This was for me to be able to test the code. In the final version lets change it back to the default.

File: pkg/ibm/vpc_collection.go [hunk 2]
```diff
@@ -222,19 +343,41 @@ func getLoadBalancers(vpcService *vpcv1.VpcV1) ([]*datamodel.LoadBalancer, error
...
 		}
 
-		res[i] = datamodel.NewLoadBalancer(&lbCollection.LoadBalancers[i], listeners, pools)
+		res[i] = datamodel.NewLoadBalancer(&loadBalancers[i], listeners, pools)
 	}
 
 	return res, nil
 }
+
+func getPoolMembers(vpcService *vpcv1.VpcV1, lbID *string, vpcPool *vpcv1.LoadBalancerPool) (datamodel.LoadBalancerPool, error) {
```
[the remark this answers is not in the document]
Comment (elazarg) [reply pkg/ibm/vpc_collection.go]: Done

File: pkg/ibm/vpc_collection.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (elazarg) [reply pkg/ibm/vpc_collection.go]: Changed to 50 (which IIUC is the default)

**Review by zivnevo — APPROVED** [review]
LGTM

**Pull request merged**